monitoring/bin/deploy_monitoring_cluster.sh

```diff
@@ -204,14 +205,14 @@ kubectl apply -n $MON_NS -f monitoring/monitors/logging/serviceMonitor-fluent-bi
 kubectl apply -n $MON_NS -f monitoring/monitors/logging/serviceMonitor-fluent-bit-v2.yaml
 
 # Rules
-log_info "Adding Prometheus recording rules..."
+log_verbose "Adding Prometheus recording rules..."
 for f in monitoring/rules/viya/rules-*.yaml; do
   kubectl apply -n $MON_NS -f $f
 done
 
 kubectl get prometheusrule -n $MON_NS v4m-kubernetes-apps 2>/dev/null
 if [ $? == 0 ]; then
-  log_info "Patching KubeHpaMaxedOut rule..."
+  log_verbose "Patching KubeHpaMaxedOut rule..."
   # Fixes the issue of false positives when max replicas == 1
```

Comment:
Do we want to keep the "..." at the end of this message?  We have removed them from other messages in this same script.

Reply:
I think we should only use the "..." if we have a message that confirms that the task is done.  In the example above, I'll go ahead and remove it, but for messages like the one below, I think we should keep it.

Deploy X ...
X has been deployed

What do you guys think?

Reply:
Consistency is the most important thing. Either way is fine as long as there's a guide as to when '...' is used.